package/smt.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Mon Dec  3 13:30:33 UTC 2018 - wstephenson@suse.com
+
+- Version 3.0.39
```

Comment:
Not quite 3.0.39:

https://github.com/SUSE/smt/blob/a58bcc20e2f0f2edca1f4cfcc535f6c4eb7c152c/Makefile#L1-L2

https://github.com/SUSE/smt/blob/a58bcc20e2f0f2edca1f4cfcc535f6c4eb7c152c/package/smt.spec#L19-L20

Reply:
But no worries, bumped up to 3.0.40 in #127.